Quizzler-iOS13/Model/QuizManager.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  QuizManager.swift
+//  Quizzler-iOS13
+//
+//  Created by Ronald Ngounou on 4/18/24.
+//  Copyright © 2024 The App Brewery. All rights reserved.
+//
+
+import Foundation
+
+class QuizManager{
+    
+    var questions: [ChoicesQuestions] = []
+    var questionNumber = 0
+    var score = 0
+    
+    init() {
+        let question1 = ChoicesQuestions(questionText: "What is the capital of France?", choices: ["London", "Paris", "Berlin"], correctAnswerIndex: 1, explanation: "Paris is the capital of France.")
```

Comment:
+1 to the comment, and even better to move such questions to a special class (DataSource or similar) instead, because even though it's fine to do it as part of the model itself, it's just not a scaling approach, it won't work:
- if you need to use some logic for how you fetch questions (sorting, filtering)
- if you want to retrieve questions from backend or DB

Reply:
Thank you.
I will move the questions in a special class.